src/test/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionSpec.scala

```diff
@@ -266,6 +258,92 @@ class SubmissionSpec(_system: ActorSystem) extends TestKit(_system) with FlatSpe
     }
   }
 
+  "Submission validation requests" should "report a BadRequest for an unparseable entity expression" in withWorkspaceService { workspaceService =>
+    val submissionRq = SubmissionRequest("dsde", "GoodMethodConfig", "Individual", "indiv1", Some("this.is."))
+    val vComplete = workspaceService.validateSubmission( testData.wsName, submissionRq ).asInstanceOf[RequestComplete[(StatusCode, String)]]
+    val (vStatus, vData) = vComplete.response
+    assertResult(StatusCodes.BadRequest) {
+      vStatus
+    }
+  }
+
+  it should "report a validated input and runnable workflow when given an entity expression that evaluates to a single entity" in withWorkspaceService { workspaceService =>
+    val submissionRq = SubmissionRequest("dsde", "GoodMethodConfig", "Pair", "pair1", Some("this.case"))
+    val vComplete = workspaceService.validateSubmission( testData.wsName, submissionRq ).asInstanceOf[RequestComplete[(StatusCode, SubmissionValidationReport)]]
+    val (vStatus, vData) = vComplete.response
+    assertResult(StatusCodes.OK) {
+      vStatus
+    }
+
+    assertResult(1) { vData.validEntities.length }
+    assert(vData.invalidEntities.isEmpty)
+  }
+
+  it should "report validated inputs and runnable workflows when given an entity expression that evaluates to a set of entities" in withWorkspaceService { workspaceService =>
+    val submissionRq = SubmissionRequest("dsde", "GoodMethodConfig", "SampleSet", "sset1", Some("this.samples"))
+    val vComplete = workspaceService.validateSubmission( testData.wsName, submissionRq ).asInstanceOf[RequestComplete[(StatusCode, SubmissionValidationReport)]]
+    val (vStatus, vData) = vComplete.response
+    assertResult(StatusCodes.OK) {
+      vStatus
+    }
+
+    assertResult(3) { vData.validEntities.length }
+    assert(vData.invalidEntities.isEmpty)
+  }
+
+  it should "report a BadRequest when given an entity expression that evaluates to an empty set of entiites" in withWorkspaceService { workspaceService =>
```

Comment:
says BadRequest, but 404
isnt this the same as the other empty expression test above?